src/messaging/ReliableMessageContext.cpp

```diff
@@ -344,7 +344,7 @@ CHIP_ERROR ReliableMessageContext::SendStandaloneAckMessage()
     CHIP_ERROR err = CHIP_NO_ERROR;
 
     // Allocate a buffer for the null message
-    System::PacketBufferHandle msgBuf = System::PacketBuffer::NewWithAvailableSize(0);
+    System::PacketBufferHandle msgBuf = System::PacketBufferHandle::New(0);
```

Comment:
This doesn't make it clear that all overhead is swallowed in "reserved" default argument. Suggest making the default reserved argument _explicit_ here.